vault-ui/src/pages/dashboard/UserSettingsPage.tsx

```diff
@@ -160,6 +170,42 @@ function RegisterPasskeyButton() {
   );
 }
 
+function DeletePasskeyButton({ passkeyId }: { passkeyId: string }) {
```

Comment:
just `useParams` inside this component, don't take a param

Reply:
There are no params (see pic)